msal/src/main/AndroidManifest.xml

```diff
@@ -13,16 +13,26 @@
             android:exported="false"
             android:launchMode="singleTask" />
 
+        <activity
+            android:name="com.microsoft.identity.common.internal.providers.oauth2.CurrentTaskAuthorizationActivity"
+            android:configChanges="orientation|keyboardHidden|screenSize|smallestScreenSize|screenLayout"
+            android:exported="false"
+            android:launchMode="standard" />
+
         <!-- Helper activity for displaying current broker redirect URI configuration -->
         <activity
             android:name="com.microsoft.identity.client.helper.BrokerHelperActivity"
             android:configChanges="orientation|keyboardHidden|screenSize|smallestScreenSize|screenLayout"
             android:exported="false"
-            android:launchMode="singleTask" />
+            android:launchMode="standard" />
 
         <!-- MSAL will use system webview (custom tab) to render the sign-in page, BrowserTabActivity is to get response back from System Webview. Multiple apps on the device could integrate MSAL, OS will check which BrowserTabActivity can handle the intent based on the intent filter declared, so this activity has to be exported.-->
         <activity
             android:name="com.microsoft.identity.client.BrowserTabActivity"
             android:exported="true" />
+
```

Comment:
We might want to call out explicitly somewhere what are the different between these two (BrowserTabActivity vs CurrentTaskBrowserTabActivity)

Reply:
They have a different implementation of the activity... in terms of the behaviors they perform.  I'll write up some internal documentation for this as well as some external documentation for consumers.